test/performance/benchmarks_test.go

```diff
@@ -130,7 +121,13 @@ func runTest(t *testing.T, img string, baseQPS float64, loadFactors []float64) {
 func TestBenchmarkSteadyTraffic(t *testing.T) {
 	for _, load := range loads {
 		t.Run(fmt.Sprintf("N%d", load), func(t *testing.T) {
-			runTest(t, app, float64(load), []float64{1})
+			zeroToNSteadyPacer := vegeta.SinePacer{
```

Comment:
This seems strange to use sine with +/-1 jitter. May be just use constant pacer? It seems we did that before anyway.

Reply:
`ConstantPacer` is generating burst traffic in my understanding, since it will generate 1000QPS from the very start if we set `Freq` to 1000. 
```
type ConstantPacer struct {
	Freq int           // Frequency (number of occurrences) per ...
	Per  time.Duration // Time unit, usually 1s
}
```

The reason for -1 is because of one restriction of `SinePacer` - `Amp` must be smaller than `Mean`. With this setting vegeta will generate steadily increasing QPS from 1 to 1000, which I think makes more sense for this test case.

Reply:
Well we can write our own, which will follow some curve and then pace at fixed top qps.
You can see how the sine one is implemented. If you need help with doing this, we can talk.